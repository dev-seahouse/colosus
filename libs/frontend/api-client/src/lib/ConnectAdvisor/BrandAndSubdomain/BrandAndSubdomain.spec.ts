import ConnectAdvisorBrandAndSubdomainApi from './BrandAndSubdomain';
import { describe } from 'vitest';

describe('BrandAndSubdomain', () => {
  const connectAdvisorBrandAndSubdomainApi =
    new ConnectAdvisorBrandAndSubdomainApi();

  describe('getSubdomain', () => {
    it('should return the correct response', async () => {
      const res = await connectAdvisorBrandAndSubdomainApi.getSubdomain();

      expect(res.data).toMatchSnapshot();
    });
  });

  describe('updateTradeNameAndSubdomain', () => {
    it('should return 204 status response', async () => {
      const res =
        await connectAdvisorBrandAndSubdomainApi.updateBrandAndSubdomain({
          subdomain: 'foo',
          tradeName: 'bar',
        });

      expect(res.status).toBe(204);
    });
  });
});
