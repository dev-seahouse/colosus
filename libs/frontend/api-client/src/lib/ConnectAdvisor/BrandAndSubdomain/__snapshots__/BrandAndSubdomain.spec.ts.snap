// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`BrandAndSubdomain > getSubdomain > should return the correct response 1`] = `
{
  "subdomain": "",
  "tradeName": "",
}
`;
